### PR TITLE
Collective principle

### PR DESCRIPTION
**Pull request by darobin**
I kept it at a relatively high level, but hopefully enough that people _can_ do something with it.

***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/pull/130.html#collective-privacy" title="Last updated on Mar 17, 2022, 10:18 PM UTC (f7dc72f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/130/a1b3a33...f7dc72f.html" title="Last updated on Mar 17, 2022, 10:18 PM UTC (f7dc72f)">Diff</a>

### CODE REVIEW

**Comment by hober** [comment]
How does this relate to cases like "share your contacts with this website," where the decision is individual but the potential privacy harm is to everybody in their contacts?

**Comment by dmarti** [comment]
Another possible source for "collective consequences": Jessica Dawson, [Microtargeting as Information Warfare](https://cyberdefensereview.army.mil/Portals/6/Documents/2021_winter_cdr/04_CDR_V6N1_Dawson.pdf?ver=_cSYLxO5k8EHPSmFziZU3g%3d%3d)

The example is national security issues resulting from individual decisions by military personnel to use network services.

File: index.html [hunk 1]
```diff
@@ -556,6 +581,29 @@
...
+There are different ways for [=people=] to become members of a group. Either they can join it
+deliberately, making it a self-constituted group such as when joining a club, or they can be
+classified into it by an external party, typically a bureaucracy or its computerised equivalent
+([[?Beyond-Individual]]). In the latter case, [=people=] may not be aware that they are being
+grouped together, and the definition of the group may not be intelligible (for instance if it is
+created from opaque machine learning techniques).
+
+Protecting group privacy can take place at two different levels. [=People=] may wish to protect
+knowledge that they are members of the group even though the existence of the group and its actions
+may be well knows (eg. membership in a dissidents movement under authoritarian rule). Conversely,
```
Comment (jyasskin) [inline index.html:595]:
```suggestion
may be well known (eg. membership in a dissidents movement under authoritarian rule). Conversely,
```

File: index.html [hunk 2]
```diff
@@ -556,6 +581,29 @@
...
+the existence of a group or at least its activities may need to be protected even in cases in which
+its members are guaranteed to remain anonymous. An example [=privacy violation=] for the latter case
```
Comment (jyasskin) [inline index.html:597]: I'd swap these two: making everything secret seems less complicated than making half the things secret.

File: index.html [hunk 3]
```diff
@@ -556,6 +581,29 @@
...
+is the fitness app Strava that did not reveal individual behaviour or identity but published heat
+maps of popular running routes. In doing so, it revealed secret US bases around which military
+personnel took frequent runs ([[?Strava-Debacle]], [[?Strava-Reveal-Military]]).
+
+When [=people=] do not know that they are members of a group, when they cannot easily find other
+members of the group so as to advocate for their rights together, or when they cannot easily
+understand why they are being categorised into a given group their ability to protect themselves
```
Comment (jyasskin) [inline index.html:604]:
```suggestion
understand why they are being categorised into a given group, their ability to protect themselves
```

File: index.html [hunk 4]
```diff
@@ -1052,6 +1100,73 @@
...
 </div>
 
+## Collective Privacy {#collective-privacy}
+
+<div class="practice">
+
+<p>
+  <span class="practicelab" id="principle-collective-privacy">
+    Groups and various forms of institutions may best protect and support autonomy by making
+    decisions that apply collectively rather than individually to either prevent or enable data
```
Comment (jyasskin) [inline index.html:1110]: They should probably make the decisions collectively, rather than just making them apply collectively?

Comment (darobin) [reply index.html:1109]: Yes! Fixed.

File: index.html [hunk 5]
```diff
@@ -1052,6 +1100,73 @@
 
 </div>
 
+## Collective Privacy {#collective-privacy}
+
+<div class="practice">
+
+<p>
+  <span class="practicelab" id="principle-collective-privacy">
+    Groups and various forms of institutions may best protect and support autonomy by making
```
[the remark this answers is not in the document]
Comment (darobin) [reply index.html:1108]: My initial thought was to state that this is possible, but I like the should version better.

File: index.html (hunk 2 above)
Comment (darobin) [reply index.html]: I don't have a preference, so happy to go with yours.

**Pull request merged**